calico/felix/futils.py

```diff
@@ -630,3 +632,35 @@ def find_longest_prefix(strs):
             else:
                 longest_prefix = longest_prefix[:shared_len]
     return longest_prefix
+
+
+def usage_count(calico_version, hostname, cluster_guid, cluster_size, cluster_type):
+    """Reports the cluster's guid, size and version to projectcalico.org.
+    Logs out of date calico versions, to the standard log file.
+
+    Keyword arguments:
+    calico_version -- the calico version
+    hostname -- the agent's hostname
+    cluster_guid -- the unique cluster identifier
+    cluster_size -- the number of felix instances
+    cluster_type -- the type of cluster
+    """
+    _log.info("usage_count")
+    try:
+        url = 'https://t736v4ek4g.execute-api.ap-northeast-1.amazonaws.com/prod/calicoVersionCheck'
+
+        urllib3.disable_warnings()
```

Comment:
Feels wrong that we should have to disable warnings, what warning are we emitting and should we fix it rather than suppress it?!